### PR TITLE
WRO-2823: Updated changelog for deprecate `agate/GridListImageItem`

### PR DESCRIPTION
**Pull request by stanca-pop-lgp**
Enact-DCO-1.0-Signed-off-by: Stanca Pop stanca.pop@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [x] Documentation was verified or is not changed
* [x] UI test was passed or is not needed
* [x] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Updated changelog for deprecated `agate/GridListImageItem` to be removed in 2.0.0.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
- Deprecate `agate/GridListImageItem` to be removed in 2.0.0.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRO-2823

### CODE REVIEW

**Comment by stanca-pop-lgp** [comment]
Build fails because of "react": "^17.0.1" and "react-dom": "^17.0.1" dependencies.

**Review by daniel-stoian-lgp — APPROVED** [review]

**Review by vladut-clapou-lgp — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ The following is a curated list of changes in the Enact agate module, newest cha
 
 ## [unreleased]
 
+### Deprecated
+
+- `agate/GridListImageItem` to be removed in 2.0.0, use `agate/ImageItem` instead
```
Comment (juwonjeong) [inline CHANGELOG.md:9]:
```suggestion
- `agate/GridListImageItem`, to be removed in 2.0.0. Use `agate/ImageItem` instead.
```

**Review by juwonjeong — APPROVED** [review]
LGTM

**Pull request merged**